spawner/run.py

```diff
@@ -32,9 +28,10 @@ def main():
     sys.stdout.flush()
     while True:
         try:
-            msg = bus_service.receive_queue_message(queueConf['queue_name'], peek_lock=False)
-            if msg.body is not None:
-                work = msg.body.decode("utf-8")
+            receiver = servicebus_client.get_queue_receiver(queueConf['queue_name'], receive_mode='peeklock')
+            received_msgs = receiver.receive_messages()
+            for msg in received_msgs:
+                work = str(b''.join(msg.body), encoding='utf-8')
```

Comment:
I see you're trying to send a `dict` in `web-server/run.py`.

As I mentioned, there're two choices:
- use `AmqpAnnotatedMessage` to send a dict, the received `message.body` will just be a `dict` object.
- convert the `dict` to a json string first, and sent it using `ServiceBusMessage`. In this way, you will get the message.body as bytes and your encoding shall work.

Reply:
I have updated by your comments.